man/bootctl.xml

```diff
@@ -377,6 +377,19 @@
         <listitem><para>Install binaries for all supported EFI architectures (this implies <option>--no-variables</option>).</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--efi-boot-option-description=</option></term>
+        <listitem><para>Description of the entry added to the firmware's boot option list. Defaults to <literal>Linux
+        Boot Manager</literal>.</para>
+
+        <para>Using the default entry name <literal>Linux Boot Manager</literal> is generally perferable as only
+        one bootloader installed to a single ESP partition should be used to boot any number of OS installations
+        found on the various disks installed in the system. Specifically distributions should not use this flag
+        to install a branded entry in the boot option list. However in situations where there is a redundant array
+        of disks, each with their own ESP partition, it can be beneficial to make it easier to identify the
```

Comment:
I'd drop the reference to RAID. Just say "However in situations whith multiple disks, each with their own ESP partition, …"

If RAID or not, if you have multiple ESPs for some reason (think: Fedora on built-in disk, and an Ubuntu on a stick), you want this.